Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Mesh_3_plugin.cpp

```diff
@@ -811,7 +814,7 @@ void Mesh_3_plugin::mesh_3(const Mesh_type mesh_type,
     {
       thread = cgal_code_mesh_3(
         polyhedrons,
-        (polylines_item == nullptr) ? plc : polylines_item->polylines,
+        protect_polylines ? polylines_item->polylines : plc,
```

Comment:
Suggestion: for the clarity of the code, `plc` should be renamed to something like `polylines_empty_container`.